### PR TITLE
Update linter and webpack configs

### PR DESCRIPTION
**Pull request by tukib**
* disable the eslint linebreak style - git handles this
* disable the eslint/typescript no non null assertion - this is useful to use and is needed to simplify some runtime code
* disable performance warnings for webpack - phaser is a big library and can't really be compartmentalised easily, and any performance issues can be checked later in development

also prettier did a reformat

### CODE REVIEW

File: server/.eslintrc.json [hunk 1]
```diff
@@ -33,49 +29,44 @@
...
-            "error",
-            { "accessibility": "explicit" }
-        ],
+        "@typescript-eslint/explicit-member-accessibility": ["error", { "accessibility": "explicit" }],
         "@typescript-eslint/explicit-module-boundary-types": ["error"],
-        "@typescript-eslint/no-inferrable-types": [
+        "@typescript-eslint/no-inferrable-types": ["error", { "ignoreParameters": true }],
+        "@typescript-eslint/naming-convention": [
             "error",
-            { "ignoreParameters": true }
```
Comment (NachoToast) [inline server/.eslintrc.json:43]: Why was this removed?

**Review by NachoToast — CHANGES_REQUESTED** [review]
Weird how prettier decided to reformat it.

File: server/.eslintrc.json (hunk 1 above)
Comment (tukib) [reply server/.eslintrc.json:43]: looks like git diff is just being weird. ignoreParameters was never under naming-convention https://github.com/NachoToast/SudoCluedo/blob/main/server/.eslintrc.json#L43

**Review by NachoToast — APPROVED** [review]
